LorafyAPI/Services/MQTTBackgroundService.cs

```diff
@@ -1,22 +1,28 @@
-﻿using uPLibrary.Networking.M2Mqtt;
+﻿using LorafyAPI.Services;
+using System.Text.Json;
+using uPLibrary.Networking.M2Mqtt;
 using uPLibrary.Networking.M2Mqtt.Messages;
 
 public class MQTTBackgroundService : BackgroundService
 {
     private readonly IConfiguration _configuration;
     private readonly ILogger<MQTTBackgroundService> _logger;
     private readonly MqttClient _client;
+    private string _json { get; set; }
+    JsonModelsParsingService _jsonModelsParsingService;
+    
 
-    static void client_MqttMsgPublishReceived(object sender, MqttMsgPublishEventArgs e)
+    void client_MqttMsgPublishReceived(object sender, MqttMsgPublishEventArgs e)
     {
-        var payload = System.Text.Encoding.Default.GetString(e.Message);
-        Console.WriteLine(payload); // TODO: Remove in the future
+        _json = System.Text.Encoding.Default.GetString(e.Message);
+        Console.WriteLine(_json);
```

Comment:
If everything is fine, please remove this Console.WriteLine. Thats why the "TODO: Remove in future" was there.